### PR TITLE
Database schema

### PR DESCRIPTION
**Pull request by annemfoley**
changed best practices from object instances to be fields within each state

### CODE REVIEW

File: backend/marina_app/migrations/0004_auto_20210304_0949.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Generated by Django 3.1.7 on 2021-03-04 14:49
+
```
Comment (jindaln) [inline backend/marina_app/migrations/0004_auto_20210304_0949.py:2]: Is there a way we can remove these auto generated files?

File: backend/marina_app/models.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    no_fee = models.BooleanField(default = False)
+    office_contact = models.BooleanField(default = False)
+    no_witness_required = models.BooleanField(default = False)
+
+    def __str__(self):
+        return self.name
+
+class IssueArea(models.Model):
+    title = models.CharField(max_length=200)
+    states = models.ManyToManyField(State)
```
Comment (jindaln) [inline backend/marina_app/models.py:27]: what is the benefit of storing a manyToMany relationship as opposed to just storing a list of states?

**Review by jindaln — COMMENTED** [review]
Mainly questions about auto-generated files and the ManyToMany models relationship between issue areas and states

File: backend/marina_app/migrations/0004_auto_20210304_0949.py (hunk 1 above)
Comment (katherinelazar) [reply backend/marina_app/migrations/0004_auto_20210304_0949.py:2]: I definitely think there is a way, honestly I'm still not sure the safe way to do that right now

**Review by jindaln — APPROVED** [review]

**Pull request merged**